orienteer-core/src/main/java/org/orienteer/core/CustomAttribute.java

```diff
@@ -54,6 +54,10 @@
 	 * Name of property which is storing link to a parent entity
 	 */
 	public static final CustomAttribute PROP_PARENT = create("orienteer.prop.parent", OType.LINK, OProperty.class, null, false, true);
+	/**
+	 * Contains list of roles separated by comma from which property is hidden
+	 */
+	public static final CustomAttribute PROP_HIDDEN_FROM = create("orienteer.prop.hidden.from", OType.LINK, OProperty.class, null, false, true);
```

Comment:
Please add TODO comment that this should be reworked after implementation in OrientDB field based security: https://github.com/orientechnologies/orientdb/issues/4157